tsconfig.json

```diff
@@ -14,6 +14,7 @@
     "noImplicitThis": true,
     "resolveJsonModule": true,
     "esModuleInterop": true,
+    "skipLibCheck": true,
```

Comment:
y tho?

Reply:
mocha and jest have a type conflict that break my build in https://github.com/OffchainLabs/retryable-tx-panel/pull/12

Reply:
to fix that you can do `yarn install --production` in the panel instead of skipLibCheck here